k8s/conf/cpu-defaults.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Is this GKE specific?

Reply:
GKE is more strict into defaults.

Reply:
It is a good defaults, enabled only for GKE, possible some other also need them.